Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Jira_Ticket_(Markdown_Table).yml

```diff
@@ -0,0 +1,1258 @@
+      project_name:
+        complex:
+          root: inputs.JiraProjectName
+      summary:
+        simple: 'Prisma Cloud Compute - compliance issues ID: ${inputs.ComplianceIssueID}'
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1060,
+          "y": 1880
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "32":
+    id: "32"
+    taskid: bfd4cf39-c9d8-4c76-866b-90f6ce11a852
+    type: regular
+    task:
+      id: bfd4cf39-c9d8-4c76-866b-90f6ce11a852
+      version: -1
+      name: Delete Context
+      description: |-
+        Delete field from context.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
+      scriptName: DeleteContext
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "12"
+    scriptarguments:
+      all:
+        simple: "yes"
+      subplaybook:
+        simple: auto
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 850,
+          "y": 90
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "26_27_yes": 0.52
+    },
+    "paper": {
+      "dimensions": {
+        "height": 2355,
+        "width": 2510,
+        "x": -210,
+        "y": -60
+      }
+    }
+  }
+inputs:
+- key: Filter
+  value: {}
+  required: false
+  description: A filter to use for extracting affected resources with a specific compliance issue id.
+  playbookInputQuery:
+- key: ComplianceIssueID
+  value: {}
+  required: false
+  description: The compliance issue id to open a ServiceNow ticket for.
+  playbookInputQuery:
+- key: ComplianceIssueDescription
+  value: {}
+  required: false
+  description: The compliance issue description.
+  playbookInputQuery:
+- key: ComplianceIssueSeverity
+  value: {}
+  required: false
+  description: The compliance issue severity.
+  playbookInputQuery:
+- key: JiraIssueTypeName
+  value: {}
+  required: false
+  description: ""
+  playbookInputQuery:
+- key: JiraProjectName
+  value: {}
+  required: false
+  description: ""
+  playbookInputQuery:
+outputs:
+- contextPath: TicketAction
+  description: What was the ticket action - create or update.
+  type: string
+- contextPath: Ticket
+  description: The ticket object which was created or updated.
+  type: string
+quiet: true
+tests:
+- PaloAltoNetworks_PrismaCloudCompute-Test
```

Comment:
Is this test relevant for all the playbooks? If not, best to provide no tests so that we are aware